src/config/dynamic-theme-fixes.config

```diff
@@ -3693,6 +3693,25 @@ article, article section, article footer, .css-1fzo8jw, .css-efdc07, article .cs
 
 ================================
 
+brack.ch
+
+CSS
+body, body > header, .popover, .popover-body {
+    background: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-text) !important;
+}
+.list-group-item, .modal-content {
+    background: var(--darkreader-neutral-background) !important;
+}
+a:not(:hover) {
```

Comment:
Still won't accept as we change colors for this parts of the page:
![obraz](https://github.com/darkreader/darkreader/assets/56877029/60b5e143-8be4-4a10-bfcf-ccec7bf4e076)

So I'm again proposing `a:not(.product__category):not(:hover) {`